sdk/src/types/block/parent.rs

```diff
@@ -31,8 +31,8 @@ impl Parents {
     /// The range representing the valid number of parents.
     pub const COUNT_RANGE: RangeInclusive<u8> = 1..=8;
 
-    /// Creates new [`Parents`].
-    pub fn new(mut inner: Vec<BlockId>) -> Result<Self, Error> {
+    /// Creates new [`Parents`] from a vec.
+    pub fn from_vec(mut inner: Vec<BlockId>) -> Result<Self, Error> {
         inner.sort_unstable_by_key(|a| a.pack_to_vec());
```

Comment:
Not related to your PR but I'm trying to remember why this is a `pack_to_vec`. I feel like I remember `sort_unstable_by_key` couldn't work with references or something.It feels a bit useless. Maybe see if you can avoid it?